plugin/core/sessions.py

```diff
@@ -599,6 +600,17 @@ def __init__(self, weaksession: 'weakref.ref[Session]') -> None:
         """
         self.weaksession = weaksession
 
+    def markdown_language_id_to_st_syntax_map(self) -> MarkdownLangMap:
+        """
+        Override this method to tweak the syntax highlighting of code blocks in popups from your language server.
+        The returned object should be a dictionary exactly in the form of mdpopup's language_map setting.
+
+        See: https://facelessuser.github.io/sublime-markdown-popups/settings/#mdpopupssublime_user_lang_map
```

Comment:
Ah, the documentation talks in context of the JSON file so it's correct with that in mind.

So we should give an example since our use is within the Python context.